specification/resources/resource-manager/Microsoft.Resources/stable/2019-10-01/resources.json

```diff
@@ -2588,7 +2588,7 @@
         "x-ms-long-running-operation": true
       }
     },
-    "/subscriptions/{subscriptionId}/resourceGroups/{sourceResourceGroupName}/validateMoveResources": {
+    "/subscriptions/{sourceSubscriptionId}/resourceGroups/{sourceResourceGroupName}/validateMoveResources": {
```

Comment:
why do we need to change the parameter name? I heard there's an issue with "subscription_id1". can you share more details?

Reply:
"subscription_id1" generated because there is a duplicated name defined in swagger.
i will close this pr according to your advice in email